doc/index.rst

```diff
@@ -205,13 +205,13 @@
                     <h4>News</h4>
                     <ul>
                     <li><em>On-going development:</em>
-                    <a href="whats_new.html"><em>What's new</em> (Changelog)</a>
+                    <a href="/dev/whats_new.html"><em>What's new</em> (Changelog)</a>
```

Comment:
I think it should be `../dev/whats_new.html`.

The best way to make sure would be if you could rebase on master. The CircleCI build was fixed and we could look at the generated doc in CircleCI.